### ISSUE TITLE
`"Would you like to restore the original configuration"` Prompt Fails with Lowercase `y`

### ISSUE THREAD

**Issue by linden** [GoogleCloudPlatform/gce-rescue#5]
when answering lowercase `y` (not uppercase `Y`) to the `"Would you like to restore the original configuration"` prompt it is not accepted and instead `Cancelled`.

example:
```sh
% gce-rescue --zone us-central1-a --name example                                                                  
The instance "example" is currently configured to boot as rescue mode since 2023-02-05 00:00:00.
Would you like to restore the original configuration ? [y/N]: y
Cancelled.
```

**Comment by halleysouza** [comment]
I was not able to replicate this issue. Is it possible that you have an extra [space] and the end of the ip "y " ? If this is the case I think we can rstrip the input, but this shouldn't be a problem when typing only "y" or "Y".

Please let me know if this is the case.

**Comment by halleysouza** [comment]
@tomerlf1 could pls have a look at this ?

**Comment by tomerlf1** [comment]
Fixed as part of commit c65b07683ca9be36d85f3c1efe559058986a1b31
